2021-12-02T19:38:14|CRITICAL|root|Unhandled exception:Traceback (most recent call last):
  File "C:\Users\RAMAN\workspace\NearField\ScopeFoundry\hardware.py", line 220, in enable_connection
    raise err
  File "C:\Users\RAMAN\workspace\NearField\ScopeFoundry\hardware.py", line 210, in enable_connection
    self.connect()
  File "C:\Users\RAMAN\workspace\NearField\ScopeFoundryHW\aist_nt_remote_spm_rpc\remote_spm_hw.py", line 49, in connect
    remote_spm = RemoteSpmRPC.RemoteSpmPrx.uncheckedCast(self.communicator.propertyToProxy('Hello.Proxy'))
  File "C:\ProgramData\Anaconda3\lib\site-packages\Ice\__init__.py", line 918, in propertyToProxy
    return self._impl.propertyToProxy(str)
Ice.ProxyParseException: exception ::Ice::ProxyParseException
{
    str = no argument provided for -e option in `hello -e:tcp -h localhost -p 54321'
}

2021-12-02T19:38:29|CRITICAL|root|Unhandled exception:Traceback (most recent call last):
  File "C:\Users\RAMAN\workspace\NearField\ScopeFoundry\hardware.py", line 220, in enable_connection
    raise err
  File "C:\Users\RAMAN\workspace\NearField\ScopeFoundry\hardware.py", line 210, in enable_connection
    self.connect()
  File "C:\Users\RAMAN\workspace\NearField\ScopeFoundryHW\aist_nt_remote_spm_rpc\remote_spm_hw.py", line 53, in connect
    remote_spm.setProperty("Ice.ACM.Client", "0")
  File "C:\Users\RAMAN\workspace\NearField\ScopeFoundryHW\aist_nt_remote_spm_rpc\RemoteSpm_ice.py", line 31, in setProperty
    return _M_RemoteSpmRPC.RemoteSpm._op_setProperty.invoke(self, ((prop, val), context))
Ice.ObjectNotExistException: exception ::Ice::ObjectNotExistException
{
    id = 
    {
        name = hello
        category = 
    }
    facet = 
    operation = setProperty
}

